iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -328,4 +334,22 @@ public static PartitionData toPartitionData(StructLike key, Types.StructType key
     }
     return data;
   }
+
+  public static void getDataAndDeleteFiles(Table table, List<DataFile> dataFiles, List<DeleteFile> deleteFiles) {
```

Comment:
code style violation, getter should return something, like Pair<dataFiles,deleteFiles>

Reply:
Fixed